tests/core/full_node/test_coin_store.py

```diff
@@ -54,39 +55,48 @@ def get_future_reward_coins(block: FullBlock) -> Tuple[Coin, Coin]:
     return pool_coin, farmer_coin
 
 
+class DBConnection:
+    async def __aenter__(self) -> DBWrapper:
+        self.db_path = Path("fndb_test.db")
+        if self.db_path.exists():
+            self.db_path.unlink()
+        self.connection = await aiosqlite.connect(self.db_path)
+        return DBWrapper(self.connection)
+
+    async def __aexit__(self, exc_t, exc_v, exc_tb):
+        await self.connection.close()
+        self.db_path.unlink()
+
+
 class TestCoinStore:
     @pytest.mark.asyncio
     @pytest.mark.parametrize("rust_checker", [True, False])
-    async def test_basic_coin_store(self, rust_checker: bool):
+    @pytest.mark.parametrize("cache_size", [0])
```

Comment:
Are we testing with and without cache? I thought we were.

Reply:
most tests are, for some reason this test is only without cache though. I didn't really consider whether that was a mistake or not. My patch doesn't change it though.